cetmix_tower_server/views/cx_tower_file_view.xml

```diff
@@ -158,7 +159,7 @@
                 <field name="full_server_path" optional="hide" />
                 <field name="sync_date_last" />
                 <field name="template_id" optional="show" />
-
+                <field name="secret_ids" widget="many2many_tags" optional="hide" />
```

Comment:
Please make this field visible only to the members of the "Manager" and "Root" groups.

Reply:
Visible != accessible